client/js/lounge.js

```diff
@@ -750,6 +763,18 @@ $(function() {
 		case "user":
 			$(".channel.active .users .user[data-name=" + $(this).data("data") + "]").click();
 			break;
+		case "user-ban":
+			socket.emit("input", {
+				target: chat.data("id"),
+				text: "/mode +b " + $(this).data("data")
+			});
+		// eslint-disable-next-line no-fallthrough
```

Comment:
I think the menu entry should reflect everything clicking it will do. Since it will both ban and kick the target user, it should read e.g. "Ban & kick".